src/mlpack/core/tree/spill_tree/spill_tree_impl.hpp

```diff
@@ -192,6 +192,76 @@ SpillTree(const SpillTree& other) :
   }
 }
 
+/**
+ *   Copy Assignment
+ * */
```

Comment:
Style error.

Reply:
I think this should be `*/` instead of the current `* */`. 
Let me know if it's incorrect.